src/utils/auth.js

```diff
@@ -0,0 +1,38 @@
+import { BASE_URL } from "./Api";
+
+// SIGNUP
+export const response = (res) => {
+  if (res.ok) {
+    return res.json();
+  }
+
+  return res;
+};
+
+//GET USER ARTICLES
+
+export async function gettingUserItems(token) {
+  const res = await fetch(`${BASE_URL}/articles`, {
+    method: "GET",
+    headers: {
+      "content-type": "application/json",
+      authorization: `Bearer ${token}`,
+    },
+  });
+  return response(res);
```

Comment:
**Could be improved**

You can make a special method for fetching and checking responses **not to duplicate** it in every request:

```js

  functon request(url, options) {
    return fetch(url, options).then(checkResponse)
  }

```

And now you can safely replace `fetch` with `request` and remove repeated checking. And you don’t need to change anything else.